extension/intents/forms/formsContentScript.js

```diff
@@ -248,6 +244,51 @@ this.dictationContentScript = (function() {
     return `${startSlice}${spaceBefore}${snippet}${spaceAfter}${endSlice}`;
   }
 
+  function insertStringContentEditable(snippet, containerElement) {
+    if (window.getSelection) {
```

Comment:
This function looks like it needs reformatting.

You don't need to test `window.getSelection` – we know this code is running in a modern Firefox browser. Also, if we can detect it doesn't work, we should just raise an exception. This code doesn't do anything, which will be confusing to users. But here I'd just remove the tests for `window.getSelection` and `sel.getRangeAt`

Reply:
Redundant tests everywhere! Sorry about that :) I have reformatted the function